libs/zauth/README.md

```diff
@@ -7,10 +7,10 @@ version            ::= "v=" Integer
 key-index          ::= "k=" Integer (> 0)
 timestamp          ::= "d=" Integer (POSIX timestamp, expiration time)
 type               ::= "t=" ("a" | "u" | "b" | "p") ; access, user, bot, provider
-tag                ::= "l=" ("s" | ""  (session or nothing))
+tag                ::= "l=" ("s" | "") ; session or nothing
```

Comment:
I'm not too familiar with this syntax; what does ";" represent?

Reply:
Some languages use `;` to indicate the a start of a comment.
I assumed it was used this way in the line 9, so so I tried to make it more consistent, because the parenthesis in line 10 seemed wrong to me